nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/navigate_through_poses_action.hpp

```diff
@@ -57,6 +76,8 @@ class NavigateThroughPosesAction : public BtActionNode<nav2_msgs::action::Naviga
       {
         BT::InputPort<geometry_msgs::msg::PoseStamped>("goals", "Destinations to plan through"),
         BT::InputPort<std::string>("behavior_tree", "Behavior tree to run"),
+        BT::OutputPort<ActionResult::_error_code_type>(
+            "navigate_through_poses_error_code", "The navigate through poses error code"),
```

Comment:
Didn't we decide all these to be `error_code_id`?

Reply:
I think there's also going to need to be documentation updates for the new ports + migration guide

Reply:
> Named navigate_to_pose_error_code + navigate_through_poses_error_code

I just copied this.